pkg/tests/end_to_end_tests/main_test.go

```diff
@@ -31,11 +31,12 @@ import (
 )
 
 var (
-	testDatabase          = flag.String("database", "tmp_db_timescale_migrate_test", "database to run integration tests on")
-	updateGoldenFiles     = flag.Bool("update", false, "update the golden files of this test")
-	useDocker             = flag.Bool("use-docker", true, "start database using a docker container")
-	useTimescaleDB        = flag.Bool("use-timescaledb", true, "use TimescaleDB")
-	postgresVersion       = flag.Int("postgres-version-major", 14, "Major version of Postgres")
+	testDatabase      = flag.String("database", "tmp_db_timescale_migrate_test", "database to run integration tests on")
+	updateGoldenFiles = flag.Bool("update", false, "update the golden files of this test")
+	useDocker         = flag.Bool("use-docker", true, "start database using a docker container")
+	useTimescaleDB    = flag.Bool("use-timescaledb", true, "use TimescaleDB")
+	// TODO (james): Replace hardcoded value
+	timescaleDockerImage  = flag.String("timescale-docker-image", "ghcr.io/timecale/dev_promscale_extension:jg-ha-dockerfile-ts2-pg14", "TimescaleDB docker image to run tests against")
```

Comment:
Yes (maybe). I would suggest that the version is named after the branch. We don't have a `main` branch in the `promscale_extension` repo. That is easily fixed.

We don't have a `master` build of the extension yet because the docker stuff hasn't been merged to `master` (or even `develop` for that matter). My plan is to gradually "upgrade" the image specified here from `develop` to `master` as we merge things in the `promscale_extension` repo.

Reply:
done